rtc_types/src/lib.rs

```diff
@@ -32,6 +32,8 @@ pub use ecall_result::*;
 pub mod byte_formats;
 pub mod enclave_messages;
 
+pub type Nonce = [u8; 24];
```

Comment:
I think it makes sense to do this when we add other libraries where this distinction will be needed.

Reply:
Well, we already have the distinction AES-GCM nonces, at the moment. I think it's fine to leave nonces unqualified when they're in a context (such as a struct or crypto function) that qualifies them, but for a stand-alone type like this one, it's not immediately clear which kind of nonce it is without digging into the surrounding code.

I think these are ones we should start disambiguating as we go: it'll probably be easier to do now rather than down the line. (I don't relish the thought of having to grep through the codebase at some point having to pick apart multiple kinds of nonces, or having bugs due to the wrong kind of nonce type being used somewhere accidentally.)

Reply:
Pulled this out into #107 since I wanted to do a follow up where we use this type alias in more places than just the token types.